src/server/services/collection.service.ts

```diff
@@ -1431,6 +1431,46 @@ export const updateCollectionItemsStatus = async ({
   if (!manage && !isOwner)
     throw throwAuthorizationError('You do not have permissions to manage contributor item status.');
 
+  const collectionMetadata = collection.metadata as CollectionMetadataSchema;
+
+  if (status === CollectionItemStatus.ACCEPTED) {
+    if (collectionMetadata?.judgesCanScoreEntries) {
+      const exists = await dbWrite.collectionItem.findFirst({
```

Comment:
Same, should it be dbRead? Just concerned with connection limit or overloading the write db